radio/src/Makefile

```diff
@@ -784,6 +784,12 @@ ifeq ($(PCB), $(filter $(PCB), SKY9X 9XRPRO AR9X))
     FLAVOUR = 9xrpro
     CPPDEFS += -DREVX
     LDSCRIPT = targets/sky9x/sam3s8c_flash.ld
+  ifeq ($(NANO), YES)
+    # use newlib-nano for linking
+    NEWLIB_NANO_FLAGS = --specs=nano.specs -u _printf_float
+    CPPDEFS += -DNANO
+  endif
+
```

Comment:
Can somebody comment on NANO for sky9x? If good, why only for 9XR-Pro and not the entire sky9x family?